docs/source/llms/rag/notebooks/index.rst

```diff
@@ -29,3 +30,19 @@ To follow along and see the sections of the notebook guide, click below:
 .. raw:: html
 
     <a href="question-generation-retrieval-evaluation.html" class="download-btn">View the Notebook</a><br/>
+
+
+Retriever Evaluation Tutorial
+-----------------------------
```

Comment:
Can we explain what this tutorial covers to entice a reader to want to click into this and discover what it is? A high-level overview is sufficient that covers the main points and the learning path 
i.e., 

``` rst
This tutorial is geared toward providing you insight into how to build a RAG application leveraging OpenAI and how to adjudicate its effectiveness using MLflow evaluate. (probably want to explain it a bit better than this example)

In this tutorial you will learn:

- How to generate an embeddings corpus from encoded documents
- How to store the processed embeddings within a Vector DB (FAISS <link to what FAISS is>)
- How to evaluate a model's capacity for retrieving relevant document matches based on a series of queries using MLflow evaluate
- etc, etc, etc...
```